crates/oxc_linter/src/lib.rs

```diff
@@ -4,7 +4,7 @@
 mod tester;
 
 mod context;
-mod rule;
+pub mod rule;
```

Comment:
Annoying to need to do this, but the integration_test is its own package, so had to expose the module.